tests/phpunit/locallib_test.php

```diff
@@ -182,6 +182,10 @@ public function should_login_redirect_testcases() {
     public function test_check_whitelisted_ip_redirect($saml, $whitelist, $expected) {
         $this->resetAfterTest();
 
+        // Setting an address here as getremoteaddr() will return default 0.0.0.0 which then is ignored by the address_in_subnet
+        // function.
+        $_SERVER['REMOTE_ADDR'] = '127.0.0.1';
```

Comment:
I'd move this into the data provider

Reply:
Moved into data provider. Additionally there is an $active flag too, review change in check_whitelisted_ip_redirect()